bin/docker-pull

```diff
@@ -3,18 +3,17 @@
 set -eu
 
 . bin/_tag.sh
+. bin/_docker.sh
 
 if [ $# -eq 0 ]; then
-    tag="$(clean_head_root_tag)"
+    tag="$(head_root_tag)"
```

Comment:
I think docker-pull should require a tag.  It doesn't make sense to pull, for instance, dirty tags.